lib/matplotlib/cbook/__init__.py

```diff
@@ -252,7 +254,7 @@ def __setstate__(self, state):
 class IgnoredKeywordWarning(UserWarning):
     """
     A class for issuing warnings about keyword arguments that will be ignored
-    by matplotlib
+    by matplotlib.
```

Comment:
capitalize

Reply:
Sure. Even though I don't get why we want the name to be capitalized. It would be simpler to keep lower case. The logo is lowercase, the package name itself is lowercase. So most times users see the name, it will be lowercase. When you look over the net, people are struggling with the capitalization (12 out of the 15 most recent stackoverflow questions use lowercase).

Reply:
I actually don't really like the capitalized form, but that was settled a while ago on a similar thread on the tracker, so let's just keep it that way...